modules/ibi-extra-partition-ibi-install-iso.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="ibi-extra-partition-ibi-install-iso_{context}"]
+= Configuring additional partitions on the target host
+
+The installation ISO creates a partition for the `/var/lib/containers` directory as part of the image-based installation process. 
+
+You can create additional partitions by using the `coreosInstallerArgs` specification. For example, in hard disks with adequate storage, you might need an additional partition for storage options, such as {lvms-first}. 
+
+[NOTE]
+====
+The `/var/lib/containers` partition requires at least 500 GB to ensure adequate disk space for precached images. You must create additional partitions with a starting position larger than the partition for `/var/lib/containers`.
+====
+
+.Procedure
+
+. Edit the `image-based-installation-config.yaml` to configure additional partitions:
++
+.Example `image-based-installation-config.yaml` file
+[source,yaml]
+----
+apiVersion: v1beta1
+kind: ImageBasedInstallationConfig
+metadata:
+  name: example-extra-partition
+seedImage: quay.io/repo-id/seed:latest
+seedVersion: "4.17.0"
+installationDisk: /dev/sda
+pullSecret: '{"auths": ...}'
+
+#...
+
+skipDiskCleanup: true <1>
+coreosInstallerArgs: 
+   - "--save-partindex" <2>
+   - "6" <3>
+ignitionConfigOverride: |
+  {
+    "ignition": {
+      "version": "3.2.0" 
+    },
+    "storage": {
+      "disks": [
+        {
+          "device": "/dev/sda", <4>
+          "partitions": [
+            {
+              "label": "storage", <5>
+              "number": 6, <6>
+              "sizeMiB": 380000, <7>
+              "startMiB": 500000 <8>
+            }
+          ]
+        }
+      ]
+    }
+  }
+
+----
+<1> Specify `true` to skip disk formatting during the installation process.
+<2> Specify this argument to preserve a partition.
+<3> The live installation ISO requires five partitions. Specify a number greater than five to identify the additional partition to preserve.
+<4> Specify the installation disk on the target host.
+<5> Specify the label for the partition.
+<6> Specify the number for the partition.
+<7> Specify the size of parition in Mbs. 
+<8> Specify the starting position on the disk in Mbs for the additional partition. You must specify a starting point larger that the partition for `var/lib/containers`.
```

Comment:
Just double-checking that these "Mbs" are indeed lowercase "b" per https://redhat-documentation.github.io/supplementary-style-guide/#Mb.

Reply:
Oh... god catch, this should be MiB actually.